cmake-modules/FolderList.cmake

```diff
@@ -2,40 +2,39 @@ macro(GetFolderList project)
+        DownloadDepVersion(sdk/core azure-core 1.5.0)
+        DownloadDepVersion(sdk/identity azure-identity 1.1.0)
     elseif(${project} STREQUAL KEYS)
-        DownloadDepVersion(sdk/core azure-core)
-        DownloadDepVersion(sdk/identity azure-identity)
+        DownloadDepVersion(sdk/core azure-core 1.5.0)
+        DownloadDepVersion(sdk/identity azure-identity 1.1.0)
     elseif(${project} STREQUAL ADMINISTRATION)
-        DownloadDepVersion(sdk/core azure-core)
-        DownloadDepVersion(sdk/identity azure-identity)
+        DownloadDepVersion(sdk/core azure-core 1.5.0)
+        DownloadDepVersion(sdk/identity azure-identity 1.1.0)
     elseif(${project} STREQUAL STORAGE_COMMON)
-        DownloadDepVersion(sdk/core azure-core)
+        DownloadDepVersion(sdk/core azure-core 1.9.0)
     elseif(${project} STREQUAL STORAGE_BLOBS)
-        DownloadDepVersion(sdk/core azure-core)
-        DownloadDepVersion(sdk/storage/azure-storage-common azure-storage-common)
+        DownloadDepVersion(sdk/core azure-core 1.8.0)
+        DownloadDepVersion(sdk/storage/azure-storage-common azure-storage-common 12.3.1)
     elseif(${project} STREQUAL STORAGE_FILES_DATALAKE)
-        DownloadDepVersion(sdk/core azure-core)
-        DownloadDepVersion(sdk/storage/azure-storage-common azure-storage-common)
-        DownloadDepVersion(sdk/storage/azure-storage-blobs azure-storage-blobs)
+        DownloadDepVersion(sdk/core azure-core 1.8.0)
+        DownloadDepVersion(sdk/storage/azure-storage-common azure-storage-common 12.3.1)
+        DownloadDepVersion(sdk/storage/azure-storage-blobs azure-storage-blobs 12.7.0)
     elseif(${project} STREQUAL STORAGE_FILES_SHARES)
-        DownloadDepVersion(sdk/core azure-core)
-        DownloadDepVersion(sdk/storage/azure-storage-common azure-storage-common)
+        DownloadDepVersion(sdk/core azure-core 1.9.0)
+        DownloadDepVersion(sdk/storage/azure-storage-common azure-storage-common 12.3.2)
     elseif(${project} STREQUAL STORAGE_QUEUES)
-        DownloadDepVersion(sdk/core azure-core)
-        DownloadDepVersion(sdk/storage/azure-storage-common azure-storage-common)
+        DownloadDepVersion(sdk/core azure-core 1.5.0)
+        DownloadDepVersion(sdk/storage/azure-storage-common azure-storage-common 12.2.3)
     elseif(${project} STREQUAL EVENTHUBS)
-        DownloadDepVersion(sdk/core azure-core)
-        DownloadDepVersion(sdk/core azure-core-amqp)
-        DownloadDepVersion(sdk/storage/azure-storage-common azure-storage-common)
-        DownloadDepVersion(sdk/storage/azure-storage-blobs azure-storage-blobs)
-    elseif(${project} STREQUAL STORAGE_FILES_DATALAKE)
+        DownloadDepVersion(sdk/core azure-core 1.10.1)
+        DownloadDepVersion(sdk/core azure-core-amqp 1.0.0-beta.1)
+        DownloadDepVersion(sdk/storage/azure-storage-common azure-storage-common 12.3.3)
+        DownloadDepVersion(sdk/storage/azure-storage-blobs azure-storage-blobs 12.8.0)
```

Comment:
I am assuming/hoping there are no merge conflicts here, but can you pull from main, just in case.

Reply:
Thank you, I updated the branch.